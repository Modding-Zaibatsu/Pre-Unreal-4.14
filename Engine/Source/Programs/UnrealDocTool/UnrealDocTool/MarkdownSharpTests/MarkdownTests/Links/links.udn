
[This is a link to www.google.com with title](http://www.google.com "Google")

[This is a link to www.google.com](http://www.google.com)
