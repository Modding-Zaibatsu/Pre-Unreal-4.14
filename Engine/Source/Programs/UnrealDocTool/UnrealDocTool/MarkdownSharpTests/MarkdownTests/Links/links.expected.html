<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en-US">
	<head>
	</head>
	<body>
		<p><a href="http://www.google.com" title="Google" >This is a link to www.google.com with title</a></p>

		<p><a href="http://www.google.com" >This is a link to www.google.com</a></p>
	</body>
</html>
